server/graphql/v1/mutations/collectives.js

```diff
@@ -462,8 +463,19 @@ export async function archiveCollective(_, args, req) {
     return updatedCollective;
   }
 
-  // TODO: cascade deactivation to EVENTs and PROJECTs?
+  const projects = await collective.getProjects();
+  const events = await collective.getEvents();
 
+  if (projects && projects.length) {
+    projects.forEach(s => {
+      s.update({ isActive: false, deactivatedAt: Date.now(), approvedAt: null, HostCollectiveId: null });
+    });
+  }
+  if (events && events.length) {
+    events.forEach(s => {
+      s.update({ isActive: false, deactivatedAt: Date.now(), approvedAt: null, HostCollectiveId: null });
```

Comment:
We usually avoid short variables, just make it clear what it is.

```suggestion
    events.forEach(event => {
      event.update({ isActive: false, deactivatedAt: Date.now(), approvedAt: null, HostCollectiveId: null });
```

Reply:
understoop and agree. Just fixed it